config.go

```diff
@@ -80,7 +81,11 @@ func (c *Config) InitDefault(log *zap.Logger) {
 	switch c.Client {
 	case grpcClient:
```

Comment:
In the case of `grpcClient`/`httpClient` this switch just exits. I guess, that you wanted to use `fallthrough` here.

Reply:
No, the lack of fallthrough is intentional, and its the same in the code before and after my change: if `c.Client` has an ok value, i.e. `httpClient` or `grpcClient`, do nothing. I can add a `// do nothing` comment to the two cases for clarity

Reply:
Why do we need to use these cases then if the real case is only an empty string? Why not just if->else? I don't think the switch branch prediction would matter here 😃